airbyte-cdk/python/unit_tests/sources/file_based/scenarios/validation_policy_scenarios.py

```diff
@@ -12,24 +12,24 @@
             "a.csv": {  # The records in this file do not conform to the schema
                 "contents": [
                     ("col1", "col2"),
-                    ("val_a_11", "val_a_21"),
-                    ("val_a_12", "val_a_22"),
+                    ("val_a_11", "1"),
```

Comment:
It feels weird that so many tests related to validation policies had to be updated. Even more, it seems like all of this is unit tested. What is the added of having those tests? 

Furthermore, it was quite troublesome for me to investigate because:
* Error is ofter "4 != 5" which doesn't say much about what is the actual error
* For some reason, the line of the assertion that fails isn't shown properly in my IDE which leads to even more question as to which line "4 != 5" is related
* When the is a specific scenario that is failing, I wasn't able to trigger only this scenario specifically